template.md

```diff
@@ -785,6 +785,7 @@ Name | Description | GitHub Activity
 * https://www.codacy.com/blog/
 * [Scala Times](https://scalatimes.com/) - Weekly newsletter about scala
 * [47 Degrees](https://www.47deg.com/blog/) - Functional Programming news, updates, and more.
+* [Explore Scala](https://kandi.openweaver.com/explore/scala) - FunctionalA curated list of Scala libraries, authors, kits, tutorials & learning resources.
```

Comment:
Typo? `FunctionalA`

Reply:
Yes its a typo. Apologies.

Reply:
Wanted to mention as 'A curated list of Scala libraries, authors, kits, tutorials & learning resources.'